### PR TITLE
Fail2ban package

### PR DESCRIPTION
**Pull request by ololyay**

### CODE REVIEW

**Comment by ololyay** [comment]
review plz

**Comment by rilian** [comment]
add it to separate security package

like utils package is done

**Comment by tilast** [comment]
wow nice

**Comment by darthrevan** [comment]
:ship:

File: packages/security.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package :security do
+  description 'Security features: fail2ban daemon, iptables rules'
+  
+  apt 'fail2ban iptables-persistent' do
+    post :install, [
+      'sudo cp /etc/fail2ban/jail.conf /etc/fail2ban/jail.local',
+      "sed -i 's/bantime  = 600/bantime = 86400/g' /etc/fail2ban/jail.local",
+      '/etc/init.d/fail2ban restart'
```
Comment (bambycha) [inline packages/security.rb]: btw, best practice to prefer upstart `service` cmd instead of direct invoke `/etc/init.d` scripts.

Comment (rilian) [inline packages/security.rb]: agree

**Comment by rilian** [comment]
generally ok beside @bambycha 's comment

**Comment by rilian** [comment]
please post an instruction here so that we can copypaste and run on all servers

**Comment by ololyay** [comment]
@rilian 
set add servers only with `security` package in  `nodes.yml` file like
```yml
server_name_1:
  enabled: true
  ip: IP_HERE
  swap_size: 2
  packages:
    security: ~
  root_user: deploy
  root_password: some_pass_if_needed
  deployer_user: deploy
server_name_2:
  enabled: true
  ip: IP_HERE
  swap_size: 2
  packages:
    security: ~
  root_user: deploy
  root_password: some_pass_if_needed
  deployer_user: deploy
server_name_3:
 ...
```
and then run `bundle exec ruby provision.rb`

**Pull request merged**